src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1252,6 +1252,13 @@
   (jdbc/do-commands
     "create index resource_params_cache_parameters_idx on resource_params_cache using gin (parameters)"))
 
+(defn add-job-id []
+  (jdbc/do-commands
+    "alter table reports add column job_id text default null"
+    "alter table catalogs add column job_id text default null"
+    "create index reports_job_id_idx on reports(job_id)"
```

Comment:
Is there any advantage here to making these partial indexes with a 'not null' condition?